src/appshell/internal/applicationactioncontroller.cpp

```diff
@@ -229,3 +230,9 @@ void ApplicationActionController::checkForUpdate()
 
     interactive()->info(trc("appshell", "No update available"), "");
 }
+
+bool ApplicationActionController::canReceiveAction(const mu::actions::ActionCode& code) const
+{
+    auto focus = QGuiApplication::focusWindow();
+    return !focus || focus->modality() == Qt::WindowModality::NonModal;
+}
```

Comment:
I don't think there are any commands this controller can handle that make sense when there's a modal popup shown